examples/resource_lacework_integration_gcp_al_pub_sub/main.tf

```diff
@@ -0,0 +1,96 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+variable "name" {
+  type    = string
+  default = "GCP pub sub audit log integration example"
+}
+
+variable "client_id" {
+  type    = string
+  default = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+}
+
+variable "client_email" {
+  type    = string
+  default = "gcp-al-pub-sub-test@techally-test.iam.gserviceaccount.com"
+}
+
+variable "private_key_id" {
+  type    = string
+  default = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+}
+
+variable "private_key" {
+  type    = string
+  default = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+}
+
+variable "integration_type" {
+  type    = string
+  default = "PROJECT"
+}
+
+variable "project_id" {
+  type    = string
+  default = "example-project-id"
+}
+
+variable "subscription" {
```

Comment:
```suggestion
variable "subscription_id" {
```

Reply:
all references to `subscription` have been updated to `subscription_id` throughout the code